lib/polo/sql_translator.rb

```diff
@@ -133,14 +133,33 @@ def raw_sql_from_record(record)
       end
     end
 
+    # Internal: Returns an object's attribute definitions along with
+    # their set values (for Rails 6.0).
+    module ActiveRecordSix
+      def raw_sql_from_record(record)
+        values = record.send(:attributes_with_values, record.class.column_names)
```

Comment:
Actually the alternate for `attributes_with_values_for_create` should be `record.send(:attributes_with_values, record.send(:attributes_for_create, attribute_names))`